### PR TITLE
Make suggestions clickable in iOS (Safari)

### PR DESCRIPTION
**Pull request by vermaxik**
These changes should fix an issue with suggestions click in iOS #37

### CODE REVIEW

File: lib/diff_web/templates/search/search.html.leex [hunk 1]
```diff
@@ -15,13 +15,13 @@
...
         <%= for suggestion <- @suggestions do %>
-          <span class="suggestion" phx-click="search_<%= suggestion %>"><%= suggestion %></span>
+          <span class="suggestion" phx-click="search_<%= suggestion %>" onclick=""><%= suggestion %></span>
         <% end %>
       <% end %>
     </div>
   </form>
   <%= if length(@releases) == 1 do %>
-    The package only has one version so there's nothing to diff with.
+    The package "<%= @query %>" has only one version so there's nothing to diff with.
```
Comment (joladev) [inline lib/diff_web/templates/search/search.html.leex]: Could we skip this part? It doesn't help make suggestions clickable in iOS :)

Comment (vermaxik) [reply lib/diff_web/templates/search/search.html.leex]: Sure, changed. 
P.S. But without explicit package name it was confusing to me.

Comment (joladev) [reply lib/diff_web/templates/search/search.html.leex]: In that case, I suggest adding a separate issue or PR for it :)

**Comment by ericmj** [comment]
Thank you!

**Pull request merged**